templates/go/model_simple.mustache

```diff
@@ -120,7 +120,7 @@ func New{{classname}}WithDefaults() *{{classname}} {
 // Deprecated
 {{/deprecated}}
 func (o *{{classname}}) Get{{name}}() *{{#isNumber}}float64{{/isNumber}}{{#isFloat}}float64{{/isFloat}}{{#isDouble}}float64{{/isDouble}}{{#isInteger}}int64{{/isInteger}}{{#isLong}}int64{{/isLong}}{{^isNumeric}}{{vendorExtensions.x-go-base-type}}{{/isNumeric}} {
-	if o == nil{{#isNullable}}{{^vendorExtensions.x-golang-is-container}} || o.{{name}}.Get() == nil{{/vendorExtensions.x-golang-is-container}}{{/isNullable}} {
+	if o == nil{{#isNullable}}|| IsNil(o.{{name}}) {{^vendorExtensions.x-golang-is-container}} || o.{{name}}.Get() == nil{{/vendorExtensions.x-golang-is-container}}{{/isNullable}} {
```

Comment:
```suggestion
	if o == nil || IsNil(o.{{name}}){{#isNullable}}{{^vendorExtensions.x-golang-is-container}} || o.{{name}}.Get() == nil{{/vendorExtensions.x-golang-is-container}}{{/isNullable}} {
```

Shouldn't it be like this? Similar to what we discussed, since we are using pointers for all fields, we should check if it is nil even for non nullable fields

Reply:
Removing the null checks broke other generated structs down the line. I'll have to check and fix this as well. The generated code is a little brittle